docker/local-universe/README.md

```diff
@@ -1,31 +1,38 @@
 
 ## Using
 
-1. If you're using the latest stable release of DC/OS, then download the [local-universe](https://downloads.mesosphere.com/universe/public/local-universe.tar.gz) container to each of your masters. Otherwise, you'll need to build the container for your DC/OS version (see the "Building Your Own" section below) and copy it to each of your masters.
+1. If you're using the latest stable release of DC/OS, then download the [local-universe]
+(https://downloads.mesosphere.com/universe/public/local-universe.tar.gz) container to each of your
+masters. Otherwise, you'll need to build the container for your DC/OS version (see the "Building
+Your Own" section below) and copy it to each of your masters.
```

Comment:
@jsancio it looks like most of these changes are formatting only. Could you add a comment to the places where the content actually changed?